### ISSUE TITLE
Site logo in header has link style border

### ISSUE THREAD

**Issue by Feng-Shui** [govCMS/ui-kit-base-theme#17]
The linked site logo in the site header is getting link styles which looks odd, screenshot below.

ui_kit.css is giving us:

```
header[role='banner'] a {
    color: #313131;
    text-decoration: none;
    outline: none;
    border-bottom: solid 1px #5bcbe3;
    transition-timing-function: ease-in-out;
}
```

Although on dta.gov.au (almost identical styles) there's no border, probably because the markup is different (the logo is the background image on a `<div>`).  We might need to look at some markup changes.

<img width="255" alt="screen shot 2016-12-15 at 9 14 03 am" src="https://cloud.githubusercontent.com/assets/758574/21203501/d86cd1d8-c2a6-11e6-9b12-40b12ac77f94.png">

**Comment by Feng-Shui** [comment]
@DJMyles suggest we use a copy of the svg logo instead of the png version.

We might have to preprocess the `$logo` var in `uikit_base_prepreocess_page()` to replace the logo.png version.

https://www.govcms.gov.au/sites/g/files/net2111/themes/site/govcms-theme/img/govcms.svg

**Comment by klepas** [comment]
We had the same issue with the new design guide recently, and we just applied an override to unset the anchor styling. (:

**Comment by DJMyles** [comment]
@klepas yeah that's pretty much exactly what we did :)

At the moment i think this one is still an open issue for other reasons more related to easy UI access to the logo.

We also had a similar issue with the contextual link styles within Drupal (the little cog that appears for authenticated users that allows them to modify elements like blocks / menus etc). The contextual markup is rendered by the module WITHIN other elements like ` .site-nav__wrapper ` so we had to pick that apart and do some overrides (which varied for `footer[role='contentinfo']` ,
`header[role='banner']` and `main`. We also had to change: 

` .js .nav-collapse {
   overflow: hidden;
 } `

to: 

` .js .nav-collapse {
   overflow: visible;
 } `

so that the menus themselves within contextual links worked.
